NetCasbin/CoreEnforcer.cs

```diff
@@ -170,6 +176,17 @@ public void SetEffector(IEffector effector)
             _effector = effector;
         }
 
+#if !NET45
+        /// <summary>
+        /// Sets an enforce cache.
+        /// </summary>
+        /// <param name="enforceCache"></param>
+        public void SetEnforceCache(IEnforceCache enforceCache)
+        {
+            EnforceCache = enforceCache;
```

Comment:
I think it is not suitable for automatic enable.
Because, SetEnforceCache is a different operation to enable the cache, and the enable cache is an unsafe operation (because we do not auto clean cache, It will return a not correct result when the policy changed).